apps/deriv-hk/styles.css

```diff
@@ -4,4 +4,9 @@
 
 @import '@deriv/quill-design/dist/quill-design.css';
 @import '@deriv/quill-design/quill-tailwind/styles.css';
-@import '@deriv/quill-design/quill-tailwind/fonts.css';
\ No newline at end of file
+@import '@deriv/quill-design/quill-tailwind/fonts.css';
+
+
```

Comment:
can you please add a `TODO` comment, explaining why we're adding this and we should look for better solution for this? 
thanks thanks 🙏